stko/molecular/conversion/utilities.py

```diff
@@ -0,0 +1,156 @@
+"""
+Utilities for molecule conversion.
+
+"""
+
+import logging
+import numpy as np
+from ase.io import write
+from .ase import ASEConverter
+
+from ...utilities import periodic_table
+
+logger = logging.getLogger(__name__)
+
+
+def write_cif(mol, unit_cell, filename):
+    """
+    Write CIF from stk.Molecule and .Cell in P1 symmetry.
+
+    This function uses the ASE interface to write a CIF.
+
+    """
+
+    # Convert stk Molecule object into ase Atoms object.
+    ase_atoms = ASEConverter().from_stk(
+        molecule=mol,
+        unit_cell=unit_cell,
+    )
+
+    write(images=ase_atoms, filename=filename)
+
+
+def with_structure_from_periodic_turbomole(mol, path):
+    """
+    Return a clone, with its structure taken from a Turbomole file.
+
+    Note that coordinates in ``.coord`` files are given in Bohr.
+
+    Parameters
+    ----------
+    path : :class:`str`
+        The full path of the ``.coord`` file from which the
+        structure should be updated.
+
+    Returns
+    -------
+    :class:`.Molecule`
+        A clone with atomic positions found in `path`.
+
+    Raises
+    ------
+    :class:`RuntimeError`
+        If the number of atoms in the file does not match the
+        number of atoms in the molecule or if atom elements in the
+        file do not agree with the atom elements in the molecule.
+
+    :class:`RuntimeError`
+        If the the Turbomole file has coordinates defined as fractional
+        based on a unit cell.
+
+    """
+
+    bohr_to_ang = 0.5291772105638411
```

Comment:
Maybe we should make a file full of constants to access them globally.

Reply:
Unnecessary, there are Python packages for doing this which we can implement if we do many conversions.